.kokoro/release/publish_javadoc11.sh

```diff
@@ -48,16 +41,18 @@ do
   # Proceed if module is not excluded
   if [[ ! "${excluded_modules[*]}" =~ $module ]]; then
     pushd $module
-    # Extract Cloud RAD module name
-    NAME=$(jq -r '.distribution_name' .repo-metadata.json | cut -d ':' -f 2)
```

Comment:
The changes in this .kokoro/release/publish_javadoc11.sh is irrelevant to diff-tool, right?